### DISCUSSION TITLE
Error Running yarn After Setting Up Local Development Environment

### DISCUSSION THREAD

**Discussion by usersdfg** [RocketChat/Rocket.Chat#32232] [General]
I'm setting up a local development environment for Rocket.Chat and following the official setup instructions. I've installed all prerequisites, including Node 14.x, Meteor, and Yarn, and I have successfully cloned the Rocket.Chat repository. However, I'm encountering an issue when I try to run the yarn command to install dependencies.

Steps I followed:

Installed Node 14.x using Volta.
Installed Meteor as specified in the [Meteor installation guide](https://www.meteor.com/developers/install).
Installed Yarn following the instructions on the [Yarn website](https://yarnpkg.com/getting-started/install).
Cloned the Rocket.Chat repository from [GitHub](https://github.com/RocketChat/Rocket.Chat.git).
Ran yarn to install dependencies.

Error Message:

![Screenshot 2024-04-17 142053](https://github.com/RocketChat/Rocket.Chat/assets/161456407/fb953c05-e87b-4fc6-ba29-b62649efb9f6)

Could someone please help me understand what might be going wrong here? Any suggestions on how to resolve this error would be greatly appreciated.

Thank you!

**Comment by d-gubert** [comment]
The error message points out 2 log files that might put you in the right direction to solve the issue, likely an error building the event-loop-stats and isolated-vm packages from NPM.
